src/WorkItems/WorkItemModel.cs

```diff
@@ -4,37 +4,41 @@
 using System;
 using System.Collections.Generic;
 
-namespace Microsoft.WorkItemFiling
+namespace Microsoft.WorkItems
 {
-    /// <summary>
-    /// Describes a work item to be filed.
-    /// </summary>
     public class WorkItemModel
     {
-        public string AccountorOrganization { get; set; }
-     
-        public string ProjectOrRepository { get; set; }
+        // TODO: Provide a meaningful representation of work item state in the model.
+        //       https://github.com/microsoft/sarif-sdk/issues/1758
+
+        // TODO: provide a meaningful unique identifier for work items, that allows
+        //       consumers to locate filed items again in the future.
+        // 
+
+        public string OwnerOrAccount { get; set; }
```

Comment:
I assume for this and all the other "Or" names, this is the most common ADO or Github name for the field?  Do we want to just unify on a single internal name?  Other wise, if feels like we just add another "or" if we decide to support Jira or another store eventually.  It feels like each field should have a specific, not implementation determined meaning.

Reply:
I agree with you, and -- it was months ago and I can't remember now -- it might even have been me that came up with these names in the first place. Fun fact: the SARIF spec ran info this issue more than once, when deciding what terminology to use for source control concepts like (what Git calls) "repo" and "commit".

---
In reply to: [381012039](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381012039) [](ancestors = 381012039)

Reply:
I intended to add a comment on this providing some explanation. The convention is that these property names reflect both the GH and ADO names, if they are not in exact agreement. The GH term always is first. So 'LabelsOrTags'. Now, this is a weird convention that will surprise most people. However, it ensures that any particular host user will find the term they understand. If we pick one host convention over another, half the team is not surprised by our convention and understands what we're referring to. The other half is out of luck. :)

btw - if this convention survives, it needs to be documented in the class summary comment (pending). We could consider expanding names to include host platform identifiers, e.g., 'GitHubLabelsOrAzureDevOpsTags'. Eep. Another tool: we could create facades on top of the core work item model (e.g., GitHubWorkItemModel) that renames things more precisely. Even if we do, consumers (such as the plugins that operate against the core model) may still need to consume a host-agnostic view.

Another issue raised by @chinarosesz is to consider what happens if/when we expand for JIRA support. This topic could be useful to discuss in stand-up parking lot, I'm guessing there's a bit of depth to it. :)

---
In reply to: [381021941](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381021941) [](ancestors = 381021941,381012039)